ambient.d.ts

```diff
@@ -0,0 +1 @@
+declare module "redis3";
```

Comment:
Don't think we need this, we just use "redis" as redis 3 now

Reply:
true